multimedia/mjpg-streamer/Makefile

```diff
@@ -1,6 +1,4 @@
 #
-# Copyright (C) 2006-2015 OpenWrt.org
-#
```

Comment:
not sure you can remove Copyright line without a bit of checking / justification